### DISCUSSION TITLE
No reopened files at start

### DISCUSSION THREAD

**Discussion by Boupaul** [WinEdt-Team/WinEdt-Support#69] [1.1 General Help and Discussion]
Hello,
Since a while, when I launch WinEdt, it doesn't reopen the previous files I was working on, which is undesired. There is no -V switch in the command line (accessed by Regedit or Configuration Wizard), the item RESTORE_OPENED_FILES=1 is present in the Project Manager (as well as REOPEN_LAST_PROJECT=1).
If I use "Delete User Profile" in the Configuration Wizard, it reopens a session at start but a session  many  months old (the last session stored I guess) and the "Rebuilt ..." command gets me back to the undesired behaviour.
Have any one an idea ?

**Comment by Alex-Simonic** [comment]
Project Menu -> Project Manager. What do you see as the Project File?

**Reply by Boupaul** [reply]
I see : \<temporary project\>

**Reply by Alex-Simonic** [reply]
This is because you closed the Project using the Close Project command. In the Recent Project List (at the bottom of the project menu) click on the Default Project (or any other project that you created -- probably you were only using the Default Project). Now it will work as before. I am not sure about your deleting the User Profile. WinEdt needs write access to its local data folder or else it cannot "remember" your opened files (and other settings).

**Reply by Boupaul** [reply]
Perfect ! And the Default Project contained all the old files I spoke about ...
I've learned something.
Thanks !
